Tools/machines/polaris-alcf/polaris_gpu_warpx.profile.example

```diff
@@ -0,0 +1,51 @@
+# Set the project name
+export proj=""  # change me!
+
+# swap to the Milan cray package
+# module swap craype-x86-rome craype-x86-milan
+
+# required dependencies
+module load cmake/3.23.2
+module load cudatoolkit-standalone
```

Comment:
Maybe add the version for slightly more explicit stability and self-documentation, if they encode it:
```suggestion
module load cudatoolkit-standalone/...
```